mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -663,6 +734,16 @@ func.func @masked_float_add_outerprod(%arg0: vector<2xf32>, %arg1: f32, %arg2: v
 // CHECK:           %[[VAL_8:.*]] = llvm.intr.fmuladd(%[[VAL_0]], %{{.*}}, %[[VAL_2]])  : (vector<2xf32>, vector<2xf32>, vector<2xf32>) -> vector<2xf32>
 // CHECK:           %[[VAL_9:.*]] = arith.select %[[VAL_3]], %[[VAL_8]], %[[VAL_2]] : vector<2xi1>, vector<2xf32>
 
+func.func @masked_float_add_outerprod_scalable(%arg0: vector<[2]xf32>, %arg1: f32, %arg2: vector<[2]xf32>, %m: vector<[2]xi1>) -> vector<[2]xf32> {
```

Comment:
Is renaming `arg[0-9]*` to `vec_[0-9]*`, ... in your future plans ?

Reply:
This file consistently uses `%arg0` and `%arg1`, so renaming would be just extra noise. In the other files, we were mixing different styles, so I've renamed everything (to `%vec`) for consistency. But, IMHO, the actual name isn't as important as consistency 😅